spec/features/user_can_read_email_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+require "pry"
+
+
+feature 'User can' do
+    let(:user) { FactoryBot.create(:user) }
+    before do
+        login_as(user, scope: :user)
+        visit user_session_path
+    end
+
+    it 'display the inbox button' do
+        expect(page).to have_content 'Inbox'
+    end
+
+    it 'navigate to inbox' do
+        click_on 'Inbox'
+        expect(page).to have_content 'Sent'
+        expect(page).to have_content 'Trash'
+    end
+end
```

Comment:
This test is a little bit 'kaka på kaka', you are in a way testing this functionality when reading the email.

Reply:
Thanks- we realised this test was superfluous but left it in so you could see our thought process in writing a sort of f-bot sanity check test.